code/datums/uplink/highly visible and dangerous weapons.dm

```diff
@@ -16,7 +16,7 @@
 
 /datum/uplink_item/item/visible_weapons/forcegloves
 	name = "Force Gloves"
-	item_cost = 8
+	item_cost = 5
```

Comment:
Hmm... I would say that if we kept the damage multiplier at 2.5, the cost should be at least 10-14 TC. But since we reduce damage multiplier hence we need to reduce price as if we bumped it to 10-14 TC. So 8-10 is still reasonable price for 1.5 damage multiplier, what do you think?

Reply:
The thing I'm taking into consideration is removing a substantial amount of burst damage from some melee weapons (which might even be considered necessary since we've plenty of melee weapons that hit between the 15-30 force benchmarks of being considered strong).

Having a limb taken off in a single blow is not very fun, especially since weapons with 25-30 force get amplified to an amount that makes them cut off the lower torso from people's body. If you lose your lower torso you die instantly, and the lower torso is easier to hit than the head.

I can probably tune the cost again, and maybe settle the damage mult to 1.75, so that force gloves are still good but not overtuned.

Reply:
My point was that the price you set for 1.5 was too low for benefit of it, I think it should have been 8-10 for 1.5. If you are gonna make it 1.75 then it should be 9-12 TC.